cypress/cypress/integration/setup.spec.ts

```diff
@@ -131,27 +203,25 @@ describe("Setup", () => {
       cy.contains("It's nice to meet you, Clinton Anderson!");
       cy.contains("Let's get started setting up your new mentor.");
     });
-    cy.get("[data-cy=next-btn]").should("not.be.disabled");
-    cy.get("[data-cy=back-btn]").should("be.disabled");
-    cy.get("[data-cy=done-btn]").should("be.disabled");
-    cy.get("[data-cy=radio-0]").should(
-      "have.css",
-      "color",
-      "rgb(27, 106, 156)"
-    );
+    cy.get("[data-cy=radio]")
+      .eq(0)
+      .should("have.css", "color", "rgb(0, 128, 0)");
```

Comment:
generally want to shy away from testing explicit colors. Is ok if we have a small number of instances